apps/discovery_streams/lib/discovery-streams/stream/source_handler.ex

```diff
@@ -38,7 +29,11 @@ defmodule DiscoveryStreams.Stream.SourceHandler do
     :ok
   end
 
-  def send_to_dlq(dead_letters, _context) do
+  defp log_message(message, context) do
```

Comment:
As long as this works good i say dont change it but i wonder, since functions are executed inside out, if its not better to wrap the log statement in a config level check instead of doing logger.debug.  Maybe this works diff since its a macro /shrug